README.md

```diff
@@ -17,59 +21,495 @@ kubectl apply -k 'github.com/howardjohn/service-apis-demo'
 kubectl wait --for=condition=Available deployment --all --timeout=120s -n istio-system
 ```
 
-### Setup variables
+#### Tutorial Variables
 
 LoadBalancer
+
 ```shell
 export INGRESS_HOST=$(kubectl -n istio-system get service istio -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 export INGRESS_PORT=$(kubectl -n istio-system get service istio -o jsonpath='{.spec.ports[?(@.name=="http")].port}')
 export INGRESS_PORT_HTTPS=$(kubectl -n istio-system get service istio -o jsonpath='{.spec.ports[?(@.name=="https")].port}')
 ```
 
 NodePort
+
 ```shell
 export INGRESS_HOST=$(kubectl -n istio-system get pod -lapp=istio -o jsonpath='{.items[0].status.hostIP}')
 export INGRESS_PORT=$(kubectl -n istio-system get svc istio -ojsonpath='{.spec.ports[?(@.name=="http")].nodePort}')
 export INGRESS_PORT_HTTPS=$(kubectl -n istio-system get svc istio -ojsonpath='{.spec.ports[?(@.name=="https")].nodePort}')
 ```
 
-## Simple Example
 
-This deploys a single deployment and Service, with a simple route exposing it over HTTP and HTTPS. A demo self signed certificate is provided.
 
-```shell
-kubectl apply -k 'github.com/howardjohn/service-apis-demo/httpbin'
-kubectl wait --for=condition=Available deployment --all --timeout=120s -n service-apis-demo
+## Gateway Tutorial
+
+The following tutorial explores different resources of the Gateway API. It goes through four different examples of Gateway that have increasing complexity to demonstrate aspects of the API.
+
+- [Single Service Gateway](./single-service-gateway) shows the simplest possible Gateway which matches all traffic and sends it to a single Service.
+- [Multi-Service Gateway](./multi-service-gateway) shows a very typical Gateway deployment where an HTTPRoute with multiple routing rules matches and routes traffic to a group of different Services.
+- [Multi-Route Gateway](./multi-route-gateway) shows how a Gateway can be shared by multiple HTTPRoute resources to enable shared infrastructure between many independent groups of users.
+- [Traffic Split](./traffic-split) demonstrates how traffic can be gracefully split between multiple Services to achieve blue-green traffic deployments.
+
+Pull this repo to your local environment. All commands should be run from the `/service-apis-demo` folder. Before starting the tutorial install the demo application and gateway-demo Namespace in your cluster. Switch your context to the gateway-demo Namespace for easier configuration and inspection.
```

Comment:
nit: if we are going to recommend previously to use `kubectl apply -k 'github.com/howardjohn/service-apis-demo'` should we consistently use kustomize from URL?